### DISCUSSION TITLE
Can't understand how to configure

### DISCUSSION THREAD

**Discussion by pyrrhus80** [cssnr/link-extractor#104] [Q&A]
Hi!
This extension seems like it could be exactly what I'm looking for but I'm struggling to understand how to set it up. 

I am using another extension to automatically refresh a particular web page every minute.
I am hoping to use link-extractor to find any links that contain specific words. The links need to be fully loaded into new tabs and then left until I have an opportunity to look at them. The original page needs to be continually monitored for new links that contain my specified words.

Can link-extractor do that?
I have read through the documentation but it's full of terminology that I do not understand.

**Comment by smashedr** [comment]
This feature does not exist. But it seems like it may be useful. Confirm this is what you are asking for:

> A new feature to monitor a specific tab (or tabs) for links that collects all links (respecting keyword filtering) into one results tab storing all results until you disable the collection. With the option to automatically open new links in new tabs.


I will add this feature to the to-do list and try to get a POC as soon as I can.

**Reply by pyrrhus80** [reply]
Almost but not quite.
I need any links that contain my specified keywords to be immediately opened in new tabs.
